app/src/main/java/org/stepik/android/view/course/ui/activity/CourseActivity.kt

```diff
@@ -473,4 +473,8 @@ class CourseActivity : FragmentActivityBase(), CourseView {
             ProgressHelper.dismiss(supportFragmentManager, LoadingProgressDialogFragment.TAG)
         }
     }
+
+    override fun onDismissed() {
```

Comment:
why not to use onResume?

Reply:
The dialog does not pause the parent activity, so it necessary to notify it via a callback (when we open the course purchase in `InAppWebViewDialogFragment`). However, it is necessary to call `coursePresenter.handleCoursePurchasePressed()` in `onResume` for cases when the user accesses the course purchase through an external browser.